duckduckgo/duckduckgo-privacy-extension
# Block third party cookies

**Reviewer:** @kdzwinel 



## Description:

This PR adds blocking functionality for 3rd party cookies

## Steps to test this PR:
1. Visit https://privacy-test-pages.glitch.me/privacy-protections/storage-blocking
2. Click "Store data"
3. Click "Retrieve data"
4. Third party header and js cookie should fail

## Automated tests:

###### Reviewer Checklist:


###### PR Author Checklist:
